arch/x86/core/Kconfig.ia32

```diff
@@ -167,6 +167,7 @@ config X86_EXCEPTION_STACK_TRACE
 	bool
 	default y
 	depends on EXCEPTION_STACK_TRACE
+	depends on DEBUG_INFO
```

Comment:
I think a `select DEBUG_INFO` would be better here.

Reply:
Fixed